src/utils/list.js

```diff
@@ -1,42 +1,54 @@
-const getListStyleType = (listType) => {
-  switch (listType) {
-    case 'upper-roman':
-      return 'upperRoman';
-    case 'lower-roman':
-      return 'lowerRoman';
-    case 'upper-alpha':
-    case 'upper-alpha-bracket-end':
-      return 'upperLetter';
-    case 'lower-alpha':
-    case 'lower-alpha-bracket-end':
-      return 'lowerLetter';
-    case 'decimal':
-    case 'decimal-bracket':
-    default:
-      return 'decimal';
+class ListStyleBuilder {
+  // defaults is an object passed in from constants.js / numbering with the following properties:
+  // defaultOrderedListStyleType: 'decimal' (unless otherwise specified)
+  constructor(defaults) {
+    this.defaults = defaults || { defaultOrderedListStyleType: 'decimal' };
   }
-};
 
-const getListPrefixSuffix = (style, lvl) => {
-  if (!style) return `%${lvl + 1}`;
+  // eslint-disable-next-line class-methods-use-this
+  getListStyleType(listType) {
+    switch (listType) {
+      case 'upper-roman':
+        return 'upperRoman';
+      case 'lower-roman':
+        return 'lowerRoman';
+      case 'upper-alpha':
+      case 'upper-alpha-bracket-end':
+        return 'upperLetter';
+      case 'lower-alpha':
+      case 'lower-alpha-bracket-end':
+        return 'lowerLetter';
+      case 'decimal':
+      case 'decimal-bracket':
+      default:
+        return this.defaults.defaultOrderedListStyleType;
+    }
```

Comment:
this looks wrong

for case 'decimal' and 'decimal-bracket' it should be 'decimal' and for the case where its not explicitly specified, it should use `this.defaults.defaultOrderedListStyleType`

Reply:
Done and rebased